src/stable/build/utils/types.ts

```diff
@@ -8,6 +8,7 @@ export type CandidAndMethodMeta = {
 export type CandidGen = 'automatic' | 'custom' | 'http';
 
 export type CanisterConfig = {
+    type?: 'azle';
```

Comment:
There are a couple of things like this that feel like they aren't necessary (because our tests were passing before), but feel like they are more technically correct and complete. Is that why they are added or was there something about the state machine that required it?